### ISSUE TITLE
Getting the version number on command line doesn't work in Medusa

### ISSUE THREAD

**Issue by Givralix** [mgba-emu/mgba#794]
```
$ medusa-emu --version
usage: medusa-emu [option ...] file

Generic options:
  -b, --bios FILE            GBA BIOS file to use
  -c, --cheats FILE          Apply cheat codes from a file
  -C, --config OPTION=VALUE  Override config value
  -d, --debug                Use command-line debugger
  -g, --gdb                  Start GDB session (default port 2345)
  -l, --log-level N          Log level mask
  -v, --movie FILE           Play back a movie of recorded input
  -p, --patch FILE           Apply a specified patch file when running
  -s, --frameskip N          Skip every N frames
  --version                  Print version and exit

Graphics options:
  -1               1x viewport
  -2               2x viewport
  -3               3x viewport
  -4               4x viewport
  -5               5x viewport
  -6               6x viewport
  -f               Start full-screen
```
This happens in 0.1-medusa-5121-eec37230.

**Comment by endrift** [comment]
Do you know if this works in mGBA? It's probably an easy fix. I can get to it in the morning.

**Comment by Givralix** [comment]
It doesn't work in mGBA 0.5.2-3 either.

**Comment by endrift** [comment]
Figured not. Since this is undoubtedly a really simple fix I'll schedule it for 0.6.0, which is slated to come out ASAP.
